ui-options.html

```diff
@@ -0,0 +1,158 @@
+                <div class="columns small-12 medium-12 large-7">
+                    <header class="floe-header">
+                        <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
+                            <span class="floe-header-tagline">
+                                flexible learning for open education
+                            </span></a>
+                        </h1>
+                    </header>
+                </div>
+                <div class="columns small-12 medium-12 large-5 align-self-bottom">
+                    <nav class="floe-nav">
+                        <ul>
+                            <li><a href="./news/index.html">News</a></li>
+                            <li><a href="./resources.html">Resources</a></li>
+                            <li><a href="./ui-options.html">Learner UI Options</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+
+            <div id="content" class="floe-content">
+                <div class="flc-toc-tocContainer toc"> </div>
+
+                <!-- Begin UI Options Information Container -->
+                <article>
+                  <h2>About Learner / UI Options</h2>
+                  <p>
+                  Learner Options (or "UI Options") provides a way to enhance or improve website usability, flexibility, and accessibility by providing a way to customize and control aspects of a website without the need for additional software or tools. For those who design, build, or maintain websites, UI Options can also help reveal areas where the web content and structure can be changed to improve flexibility and robustness.
+                  </p>
+                  <h3>Get Learner / UI Options</h3>
+                  <p>
+                  For Wordpress: <a href="https://github.com/fluid-project/uio-wordpress-plugin">UI Options Wordpress Plugin</a>.
+                  </p>
+
+                  <p>
+                  For code projects: <a href="https://www.npmjs.com/package/infusion">using npm</a>, or <a href="https://github.com/fluid-project/infusion/releases">GitHub</a>.
+                  </p>
+                  <h3>Guides and Documentation</h3>
+                  <p>
+                  If installing Learner / UI Options using npm or from a github release, "<a href="https://docs.fluidproject.org/infusion/development/tutorial-userInterfaceOptions/UserInterfaceOptions.html">Setting Up User Interface Options</a>" will help installing and structuring your code (Note: this is not applicable if using the Wordpress plugin).
+                  </p>
+                  <p>
+                  Once Learner / UI Options is installed, <a href="https://docs.fluidproject.org/infusion/development/tutorial-userInterfaceOptions/WorkingWithUserInterfaceOptions.html">"Working with UI Options"</a> can help guide you through setting up your site's styles and markup to take advantage of Learner / UI Options.
```

Comment:
Again, maybe replace the "Learner / UI Options" instances with "Learner Options"